rllib/env/tests/test_multi_agent_episode.py

```diff
@@ -1073,7 +1099,7 @@ def test_getters(self):
         self.assertListEqual(episode_1.global_t_to_local_t["agent_5"][-2:], [99, 100])
         # Agent 5 has already died, so we need to convert back to list.
         self.assertListEqual(
-            episode_1.agent_episodes["agent_5"].rewards.tolist()[-2:],
+            list(episode_1.agent_episodes["agent_5"].rewards)[-2:],
```

Comment:
Even better:
`list(episode_1.agent_episodes["agent_5"].rewards[-2:])`

This would test the proper functioning of the `[]` syntax on the buffers additionally.